### ISSUE TITLE
Worse performance with DiverseBlock for Cifar10, ResNet18

### ISSUE THREAD

**Issue by WilhelmT** [DingXiaoH/DiverseBranchBlock#15]
Hello.

Thank you for your interesting work, and code. 

I tried using your Diverseblock in ResNet18 (according to your instructions, replacing conv+bn with diverse blocks). My code is based on https://github.com/kuangliu/pytorch-cifar. The accuracy drops from 95.4% to 95.1%. Do you have any ideas for why this is?

Thank you.

**Comment by DingXiaoH** [comment]
Hi,

It may be due to the overfitting. The baseline of 95.4% on CIFAR10 is pretty high, suggesting that the model fits the data pretty well. With DBB, it may overfit. You may change the weight decay or use dropout.
